command/genesis/polybft_params.go

```diff
@@ -29,14 +31,18 @@ const (
 	sprintSizeFlag       = "sprint-size"
 	blockTimeFlag        = "block-time"
 	validatorsFlag       = "polybft-validators"
+	bridgeFlag           = "bridge"
 
 	defaultEpochSize                  = uint64(10)
 	defaultSprintSize                 = uint64(5)
 	defaultValidatorSetSize           = 100
 	defaultBlockTime                  = 2 * time.Second
 	defaultPolyBftValidatorPrefixPath = "test-chain-"
+	defaultContractsRootFolder        = "core-contracts/artifacts/contracts"
```

Comment:
Maybe you could use this constant instead 
`contracts.ContractsRootFolder`?

Reply:
Yes, this is some leftover which is not deleted